### PR TITLE
[13699]: Scan tab animation

### LINKED ISSUE #13699: Implement Scan tab in switcher
As a user
I want to have a Scan tab in the switcher.

[Switcher](https://github.com/status-im/status-mobile/issues/13155) and [Switcher tabs](https://github.com/status-im/status-mobile/pull/13386) are being implemented separately.
Scope of the issue is to implement QR scanner screen and animations. 

### Designs
https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=451%3A18994

### Animations
https://user-images.githubusercontent.com/17097240/180041582-52f805b0-af41-44e9-aa9e-518484a1f276.mp4

Principle file https://www.notion.so/Animation-guidelines-49cbc3e7a52b4120850936ca35d959ee

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #13699 
status: WIP

Progress so far

https://user-images.githubusercontent.com/33176106/180609030-bc5014d4-6317-4c97-92f0-917678d67795.mp4

### CODE REVIEW

**Comment by ibrkhalil** [comment]
This is what it looks like now

https://user-images.githubusercontent.com/33176106/180839528-25d71d51-78a7-45e8-b683-b45cdb0bf638.mp4

File: src/quo2/components/qr_scan_animation.cljs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+         [corner :border-bottom-width :border-left-width :bottom-left]
+         [corner :border-bottom-width :border-right-width :bottom-right]]]]])])
+
+(defn preview
+  []
+  [:f>
+   (fn []
+     (let [{:keys [width]} (dimensions/window)
+           size (reanimated/use-shared-value (* width 0.4))
+           _ (react/effect! (fn []
```
Comment (flexsurfer) [inline src/quo2/components/qr_scan_animation.cljs]: you could move it inside `let`

Comment (ibrkhalil) [reply src/quo2/components/qr_scan_animation.cljs]: Moved it inside the expressions body

**Comment by Parveshdhull** [comment]
> status: WIP

Please mark PR WIP also in title, so its easier to know when PR is ready for review. `[WIP]: PR Title`

**Comment by flexsurfer** [comment]
replaced by https://github.com/status-im/status-mobile/issues/14011

**Pull request closed without merging**